plasma_framework/migrations/6_output_results.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
```

Comment:
Do you mean you would like it to output the results as it is updating, or write to a local file once its done?

Reply:
ah, as for this comment, I was just wondering about would the naming of `6_output_results.js` be good enough to indicate what result it would be. Also, as I think this script would be continuous appending, we would end up in certain time have another `n_output_results.js` again, so would it be confusing or not? (I re-think about it, probably it's not confusing, and this `output_results.js` can be a bit like a tag showing where each batch deployment start and ends.)

Reply:
I understand what you mean, I feel like we're prematurely optimizing on the naming of the file. I still don't quite get what you mean by "can be a bit like a tag showing where each batch deployment start and ends."

Reply:
> we're prematurely optimizing on the naming of the file

Agree.

> can be a bit like a tag showing where each batch deployment start and ends.

ah, sorry. I mean that keeping this extra `n_output_results.js` (or any other similar named file) we can know that how migration scripts are updated together as a batch so it is actually nice to have it like what we currently have. Sorry for the confusion. haha I am basically saying it is actually great as it is now.

for instance, `6_output_results.js` and if there is a `10_output_results.js`, then we know that 7~9 would be something that we are updating to the framework together. So it becomes the tag.

Reply:
Gotcha, but wouldn't the output from `7_x_x.js` until `9_x_x.js` clog the screen?

Reply:
oh, I was only on the file name topic. As for "write to a local file once its done" my intuition is that you're correct. If we write to file, do you have idea/plan of how we use that file?

Reply:
I was thinking of saving it to a `results.md` file and just pushing it to the repo....

Reply:
sounds nice! :)

Reply:
Just on the output_results thing - the reason we did this originally was for the CI pipeline, the scripts needed to parse the output and then do stuff with the returned addresses (e.g. set up the elixir-omg config files). @jbunce will know the details better than I.

I'm not sure if writing it to a file in the repo is a good idea, won't it cause conflicts between different deployments?
In general I think it's cleaner to keep deployment specific stuff out of the repo

Reply:
okay, I think just find one more reason to not put this in console directly as this would be printed in our `truffle test` too @@

Reply:
I agree with @kevsul here - the code and deployment artifacts should be kept separate. What's in the `output_results`? Is there anything sensitive?

Reply:
@jbunce I don't think there is anything sensitive
Following is an example output now.

```
{"plasma_framework_tx_hash":"0x1daba7c8669a06af80db8f9aa7d5c69390574d21851f707b2a45c8fe9795f7af","plasma_framework":"0x345ca3e014aaf5dca488057592ee47305d9b3e10","eth_vault":"0xb9a219631aed55ebc3d998f17c3840b7ec39c0cc","erc20_vault":"0xa9f8fef0b3df9159f1443427daa79210fceb009c","payment_exit_game":"0xa4392264a2d8c998901d10c154c91725b1bf0158","authority_address":"0xc5fdf4076b8f3a5357c5e395ab970b5b54098fef"}
```